homeassistant/components/lamarzocco/config_flow.py

```diff
@@ -165,3 +220,41 @@ async def async_step_reauth_confirm(
             )
 
         return await self.async_step_user(user_input)
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> OptionsFlow:
+        """Create the options flow."""
+        return LmOptionsFlowHandler(config_entry)
+
+
+class LmOptionsFlowHandler(OptionsFlowWithConfigEntry):
+    """Handles options flow for the component."""
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Manage the options for the custom component."""
+        if user_input:
+            self.hass.config_entries.async_update_entry(
+                self.config_entry,
+                data=self.config_entry.data | user_input,
+                options=self.options,
+            )
+            return self.async_create_entry(title="", data=user_input)
```

Comment:
Hold up


```suggestion
            return self.async_create_entry(title="", data=user_input)
```

Is enough